.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+<!--
+Thank you for filing a feature request! Please provide a short summary of the new
+you need along with any information that might be required to understand the request.
+
+If you feel like any of the provided sections are not needed, 
+feel free to delete the corresponding section and leave it unfilled.
+-->
+
+**What is your motivation. Please describe!**
+
+**Describe the solution you'd like**
+<!-- A clear and concise description of what functionality we are missing to provide and how you want it to be provided. -->
+
+**Describe alternatives you've considered**
+<!-- A clear and concise description of any alternative solutions or features you've considered (and possibly why they do not suffice) -->
+
```

Comment:
```suggestion
```
I don't know if we should ask for alternatives at this point. It will probably never be used and if one wants to have a feature implemented and problems occur, there will mostly be a discussion about it anyway.